spec/langchain/vectorsearch/pgvector_spec.rb

```diff
@@ -47,7 +47,44 @@
       it "adds texts" do
         command = subject.add_texts(texts: ["Hello World", "Hello World"])
```

Comment:
The way this was written _suggests_ that ids are optional in this method - however, from the usage I could find (for instance, https://github.com/andreibondarev/langchainrb/blob/7d409a378ab300f4641a71b74d25e5cd9593122f/lib/langchain/active_record/hooks.rb#L52-L55) it seems like this is not the case.

I'd be more than happy to enforce the existence of texts/ids in tandem, but not sure if there is an (upcoming?) use case that relies on this API.